### PR TITLE
Feature/node 16

### PR DESCRIPTION
**Pull request by creisle**
Improvements
- Add node 16 to build
- separate json-schemas from code to improve readability

New Features
- Add a maxRecords option for testing snakemake workflow locally (only load x records for any given loader)
- add config option and skip fdaSrs data by default (see #61)

BugFixes
- (cosmic): Fix typo where loaders were looking for mappingFilename which had been renamed upstream to classification
- (cosmic): Use toJSON method on variant parser to avoid posting invalid parsed properties (ex. notationType)

### CODE REVIEW

File: Snakefile [hunk 1]
```diff
@@ -18,6 +17,7 @@ COSMIC_EMAIL = config.get('cosmic_email')
 COSMIC_PASSWORD = config.get('cosmic_password')
 USE_COSMIC = COSMIC_EMAIL or COSMIC_PASSWORD
```
Comment (elewis2) [inline Snakefile:18]: is COSMIC_EMAIL now defined elsewhere?

Comment (creisle) [reply Snakefile:18]: no it hasn't changed. it is still defined in line 16

File: Snakefile [hunk 2]
```diff
@@ -135,8 +137,8 @@ rule download_cgi:
     output: f'{DATA_DIR}/cgi/cgi_biomarkers_per_variant.tsv'
     shell: dedent(f'''\
         cd {DATA_DIR}/cgi
-        wget https://www.cancergenomeinterpreter.org/data/cgi_biomarkers_latest.zip
-        unzip cgi_biomarkers_latest.zip
+        wget https://www.cancergenomeinterpreter.org/data/cgi_biomarkers_20180117.zip
+        unzip cgi_biomarkers_20180117.zip
```
Comment (elewis2) [inline Snakefile:141]: why is it being fixed at this 2018 version?

Comment (creisle) [reply Snakefile:141]: They removed the "latest" version and 2018 is the latest version they have. This source is no longer being updated as far as I know

**Review by elewis2 — APPROVED** [review]

**Pull request merged**